client/src/components/Forms/FamilyIncome/Form/index.tsx

```
@@ -10,7 +10,7 @@ export const FamilyIncomeForm: React.FC<EditFormProps> = (props) => {
   const [showRedeterminationForm, setShowRedeterminationForm] = useState(false);
   const [currentIsNew, setCurrentIsNew] = useState(false);
 
-  if (!child) return <></>;
+  if (!child || !child.family) return <></>;
```

Should we throw an error here instead?

---

I would agree with this, since family is nested within child